packages/corelight/changelog.yml

```
@@ -1,4 +1,9 @@
 # newer versions go on top
+- version: "0.2.1"
+  changes:
+    - description: Added an alert insight panel and updated the VPN connections visualization to a line chart in the security posture dashboard.
+      type: enhancement
+      link: https://github.com/elastic/integrations/pull/11998
```

```suggestion
- version: "0.3.0"
  changes:
    - description: Added an alert insight panel and updated the VPN connections visualization to a line chart in the security posture dashboard.
      type: enhancement
      link: https://github.com/elastic/integrations/pull/11998
```

Since we have new panels and an `enhancement`, I think we should increment minor